src/Concecionaria.java

```diff
@@ -1,28 +1,25 @@
-import java.text.DecimalFormat;
 
-public class Concecionaria extends vehiculo {
-    private static DecimalFormat df2= new DecimalFormat("#.###");
-    public static void main(String[] Args) {
-
-        Concecionaria p206 = new Concecionaria(); p206.marca = "Peugeot"; p206.modelo = "206";   p206.puertas = 4;          p206.precio = 200000.00;
-        Concecionaria p208 =new Concecionaria();  p208.marca="Peugeot";   p208.modelo="208";     p208.puertas= 5;           p208.precio=250000.00;
-        Concecionaria hTitan =new Concecionaria();hTitan.marca="Honda";   hTitan.modelo="Titan"; hTitan.cilindrada="125cc"; hTitan.precio=60000.00;
-        Concecionaria yYbr =new Concecionaria();  yYbr.marca="Yamaha";    yYbr.modelo="YBR";     yYbr.cilindrada="160cc";   yYbr.precio=80500.50;
+public class Concecionaria  {
 
-        System.out.println("Marca: "+p206.marca+" //"+" Modelo: "+p206.modelo+" //"+" Puertas: "+p206.puertas+" //"+" Precio: $"+p206.precio+"0");
-        System.out.println("Marca: "+p208.marca+" //"+" Modelo: "+p208.modelo+" //"+" Puertas: "+p208.puertas+" //"+" Precio: $"+p208.precio+"0");
-        System.out.println("Marca: "+hTitan.marca+" //"+" Modelo: "+hTitan.modelo+" //"+" Cilindrada: "+hTitan.cilindrada+" //"+" Precio: $"+hTitan.precio+"0");
-        System.out.println("Marca: "+yYbr.marca+" //"+" Modelo: "+yYbr.modelo+" //"+" Cilindrada: "+yYbr.cilindrada+" //"+" Precio: $"+yYbr.precio+"0");
+    public static void main(String[] Args) {
+        VhAuto p206 = new VhAuto("Peugeot","206",200000.000,4);
+        VhAuto p208 =new VhAuto("Peugeot","206",250000.00,5);
+        VhMoto hTitan =new VhMoto("Honda","Titan",60000.00,"125cc");
+        VhMoto yYbr =new VhMoto("Yamaha","YBR",80500.00,"160cc");
```

Comment:
Podrías pasar esto a algo como
```suggestion
        List<Vehiculo> vehiculos = getVehiculos();
```
y quedaría mejor

Reply:
7dbe4ec6f572a847ec5e6236eff4cfcf77143bda